appendix_meta_rules.md

```diff
@@ -194,15 +208,20 @@ action: correlation
 
 **Attribute:** rules
 
-Refers to one or multiple Sigma or Correlations rules. Allowing the user to chain multiple correlations together. A rule can be referred to by the `id` or (to be defined) `name` attribute of a Sigma rule. Latter is a human-readable name and improves the readability of correlation rules.
+Refers to one or multiple Sigma or Correlations rules.
+Allowing the user to chain multiple correlations together.
+A rule can be referred to by the `id` of a sigma rule.
+
+Latter is a human-readable name and improves the readability of correlation rules.
+In this case, the tool must be able to manage the human-to-id correspondence automatically.
```

Comment:
`Latter`? What does it reference? The name field is not introduced before.

```suggestion
A rule can be referred to by the `id` or `name` of a sigma rule.

`name` is a unique human-readable name and improves the readability of correlation rules.
In this case, the tool must be able to manage the human-to-id translation automatically and the referenced rule name has to be defined in the respective rule.
```

At least that's how I understood this the last time we talked about the topic. Can you verify @thomaspatzke ?